content/03-reference/01-tools-and-interfaces/02-prisma-client/057-null-and-undefined.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Null and undefined'
+metaTitle: 'Null and undefined (Reference)'
+metaDescription: 'How Prisma Client handles null and undefined, including a GraphQL use case.'
+preview: false
+---
+
+<TopBlock>
+
+Prisma Client differentiates between `null` and `undefined`:
+
+* `null` is a **value**
+* `undefined` means **do nothing**
+
+> **Note**: This is particularly important to account for in [a **Prisma with GraphQL context**, where `null` and `undefined` are interchangeable](#use-case-null-and-undefined-in-a-graphql-resolver).
+
+In the following example, if `emailInput` is **not** valid, the query sets `email` (a **mandatory** field) to `undefined` - which means ✔ **do not include this in the update**:
```

Comment:
I was confused that the "invalid" data is `null` (especially as this article is about null vs undefined). Not sure that is really a problem though.

Reply:
Fair  - changed to 'if `emailInput` is `null`'.